lib/jenkins_pipeline_builder/extensions/builders.rb

```diff
@@ -381,7 +381,7 @@
   builder do
```

Comment:
Actually upon second look, this builder is nested within the :checkmarx_scan builder, and I'm surprised the specs pass.

I'm going to push a change to remove this.

@dtyler